mk/Common.mk

```diff
@@ -67,6 +67,10 @@ ifndef RISCVDV_REPO
 $(error Must define a RISCVDV_REPO to use the common makefile)
 endif
 
+ifndef EMBENCH_REPO
```

Comment:
At this point I'm on not sure if I can make a good call, as I've copied the method from the existing make method for the other external repos, like RISCVDV. Maybe @strichmo can comment on this?

Reply:
Yeah, this sort of got away on us.   The question is, what are the minimum set of shell env variables that must be set?  Obviously RISCV or COREV must be set for all simulations.  Beyond that it is not clear.  We could make a good case for RISCVDV since most users will be using the ISG.  Beyond that, things like the Compliance suite and Embench suite may or may not be required.

Reply:
Let's make this a warning.

Reply:
Changed to warning